os/configuring-dns.md

```diff
@@ -0,0 +1,51 @@
+# DNS configuration
+
+By default, DNS resolution on CoreOS is handled through `/etc/resolv.conf`, which is a symlink to `/run/systemd/resolve/resolv.conf`.
+This file is managed by [systemd-resolved][systemd-resolved].
+Normally, `systemd-resolved` gets DNS IP addresses from [systemd-networkd][systemd-networkd], either via DHCP or static configuration.
+DNS IP addresses can also be set via `systemd-resolved`'s [resolved.conf][resolved.conf].
+See [Network configuration with networkd]({{site.baseurl}}/docs/os/network-config-with-networkd) for more information on `systemd-networkd`.
```

Comment:
We're rewriting these links now when they get sucked in, so you can probably make this link work for people viewing this on Github. I think it should be:

`[Network configuration with networkd](network-config-with-networkd.md)`